llama_stack/templates/experimental-post-training/build.yaml

```diff
@@ -4,10 +4,22 @@ distribution_spec:
   description: Experimental template for post training
   docker_image: null
   providers:
+    inference:
+    - inline::meta-reference
+    eval:
+    - inline::meta-reference
+    scoring:
+    - inline::basic
     post_training:
     - inline::torchtune
     datasetio:
     - remote::huggingface
     telemetry:
     - inline::meta-reference
+    agents:
```

Comment:
Might be useful to use distro_codegen.py to generate these build.yaml / run.yaml. See https://github.com/meta-llama/llama-stack/blob/c8be0bf1c92318b317352decf206855abdc5e55a/llama_stack/templates/meta-reference-gpu/meta_reference.py#L23

Reply:
good suggestion! Will add a script for auto gen yaml files and distribution template doc as fast follow up!